src/migrations/20181202100524-create-users.js

```diff
@@ -13,7 +13,7 @@ export default {
     },
     email: {
       type: Sequelize.STRING,
-      unique: true,
+      unique: false,
```

Comment:
Also, @Daymorelah, I feel each user should have a unique email address.. to eliminate situations where a user registers more than once using the same email.

```suggestion
      unique: true
```